ts/src/bitfinex.ts

```diff
@@ -898,7 +898,7 @@ export default class bitfinex extends Exchange {
         return this.parseTicker (ticker, market);
     }
 
-    parseTicker (ticker, market: Market = undefined): Ticker {
```

Comment:
@samgermain this is wrong, bitfinex/bitfinex2 returns an array

Reply:
to be fair, I wouldn't change every implementation because of an exchange like bitfinex. we can either have custom methods there (which is tricky if you use aux methods like `parseTicker`) or wrap the wrap the array on a dict and extract it inside

Reply:
> @samgermain this is wrong, bitfinex/bitfinex2 returns an array

It doesn't look like it's returning an array

## bitfinex

![](https://i.imgur.com/0Fz56dD.png)

## bitfinex2

![](https://i.imgur.com/uUyYLiw.png)